merge_request_approvals_test.go

```diff
@@ -0,0 +1,153 @@
+package gitlab
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+const exampleResponse = `{
```

Comment:
This needs a more specific name as this is to generic and would cause name collisions when used by all tests.